src/ngraph_builder.cc

```diff
@@ -444,6 +444,17 @@ static Status TranslateBinaryOp(
       });
 }
 
+static Status TranslateAllreduceOp(
+    const Node* op, const std::vector<const Tensor*>& static_input_map,
+    Builder::OpMap& ng_op_map) {
+  shared_ptr<ng::Node> ng_input;
+  TF_RETURN_IF_ERROR(GetInputNodes(ng_op_map, op, &ng_input));
+
+  SaveNgOp(ng_op_map, op->name(), make_shared<ng::op::AllReduce>(ng_input));
+  NGRAPH_VLOG(1) << "after translation";
```

Comment:
Use a lower log level.

Reply:
Also I know you were using this for debugging. I wonder if you want to keep this. If yes, please make the log more informative.

Reply:
Removed this debug comment